src/modules/client/client.service.ts

```diff
@@ -0,0 +1,83 @@
+import { BadRequestException, Injectable, NotFoundException } from '@nestjs/common';
+import { genSaltSync, hashSync } from 'bcryptjs';
+import { PageResponse } from 'src/constants/PageResponse';
+import { DeleteResult } from 'typeorm';
+import { Client } from '../../../entities/Client.entity';
+import { ClientDAO } from './client.dao';
+
+@Injectable()
+export class ClientService {
+  constructor(private readonly clientDao: ClientDAO) {}
+
+  async getClientInfoById(id: string, companyId?: string): Promise<Client> {
+    return this.clientDao.getUserInfoById(id, companyId);
+  }
+
+  async getClientByEmail(email: string, companyId?: string): Promise<Client> {
+    return this.clientDao.getUserByEmail(email, companyId);
+  }
+
+  async update(id: string, user: Client, companyId: string): Promise<Client> {
```

Comment:
@fernandogcabal please rename the var that represents the Client entity to **client**